src/DynamoCoreWpf/Views/Gallery/GalleryView.xaml

```diff
@@ -101,15 +101,31 @@
             />
 
         <!-- Close button -->
-        <dynui:ImageButton 
-            Width="86" Height="21"
-            Grid.Column="1" Grid.Row="0"
+        <TextBlock 
+            FontFamily="{StaticResource OpenSansRegular}"
+            Text="{Binding Path=CloseButton}"
```

Comment:
I have tried to create a new png.image, but I have no idea how to design a correct pixel that can fix the page . so I choose to use textbox instead.